bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -730,12 +726,12 @@ func (r *KubeadmConfigReconciler) reconcileDiscovery(ctx context.Context, cluste
 	if config.Spec.JoinConfiguration.Discovery.BootstrapToken.Token == "" {
 		remoteClient, err := r.remoteClientGetter(ctx, r.Client, util.ObjectKey(cluster), r.scheme)
 		if err != nil {
-			return err
+			return ctrl.Result{}, err
```

Comment:
Wrap error?

Reply:
Same as above, the returned error is already wrapped